src/main/drivers/flash/flash_w25q128fv.c

```diff
@@ -168,19 +153,18 @@ MMFLASH_CODE static uint8_t w25q128fv_readRegister(flashDeviceIO_t *io, uint8_t
     return in[0];
 }
 
+#ifndef PICO
```

Comment:
This should be moved out... no platform specific defines in sec/main :)

Reply:
Done